tests/prague/eip2935_historical_block_hashes_from_state/test_block_hashes.py

```diff
@@ -134,16 +132,15 @@ def test_block_hashes_history_at_transition(
                 storage=storage,
             )
 
+            code_address = pre.deploy_contract(code)
             txs.append(
                 Transaction(
-                    to=current_code_address,
+                    to=code_address,
                     gas_limit=10_000_000,
-                    nonce=next(tx_nonce),
+                    sender=sender,
```

Comment:
For cases where we only have 1 sender (eoa), could we automatically add the sender to a tx too? Such that in the default basic case we wouldn't need to add the sender field.

I think this is less useful than the above.

Reply:
I could not find a way of doing this easily without doing something hacky, so I think we could postpone this until we find a clean solution.